website/docs/basics/tips-tricks.md

```diff
@@ -67,3 +67,21 @@ class Components extends LibsComponents
 	}
 }
 ```
+
+### Internationalization (I18n) and localization (L10n)
+
+To make sure your project is translatable into other languages, you should [WordPress's i18n functions](https://codex.wordpress.org/I18n_for_WordPress_Developers) every time you output text. This process, called internationalization, is quite simple to implement by using the `__` WordPress function, which is available in both PHP and Gutenberg (in the `@wordpress/i18n` package, and is standard practice in Eightshift Frontend Libs for all text output. The `__` function accepts a string to translate and a text domain, which is used to tell WordPress which translation set to use, and returns the translated string (or original string, in case a translation doesn't exist). Note that you can provide a `sprintf`-parametrized function to `__` and use `sprintf` on its return value to populate the string with certain values. You should also load the text domain using `load_theme_textdomain`. Using the Eightshift Libs `I18n` class will do this for you by instructing WordPress to look for translations in `src/I18n/languages` with the text domain defined as your project name.
```

Comment:
```suggestion
To make sure your project is translatable into other languages, you should use the [WordPress's i18n functions](https://codex.wordpress.org/I18n_for_WordPress_Developers) every time you output text. This process, called internationalization, is quite simple to implement by using the `__` WordPress function, which is available in both PHP and Gutenberg (in the `@wordpress/i18n` package), and is standard practice in Eightshift Frontend Libs for all text output. 

The `__` function accepts a string to translate and a text-domain, which is used to tell WordPress which translation set to use and returns the translated string (or original string, in case the translation doesn't exist). Note that you can provide a `sprintf`-parametrized function to `__` and use `sprintf` on its return value to populate the string with certain values. You should also load the text domain using the `load_theme_textdomain` function. Using the Eightshift Libs `I18n` class will do this for you by instructing WordPress to look for translations in the `src/I18n/languages` folder with the text domain defined as your project name.
```